### PR TITLE
Added zaproxy RT BT env

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/Be-Secure/issues/425

Added zaproxy RT BT env 0.0.1
updated list.txt

### CODE REVIEW

**Comment by pramit-d** [comment]
@sonatype-lift ignoreall


Regards,



[Shape, circle  Description automatically generated]

Pramit Kumar Dutta

Cyber Security Analyst

OSS Security Centre Of Excellence

CRS Emerging Tech Security

Wipro Technologies, Kolkata

***@***.***

+91 8637316389 (IST)

________________________________

**Comment by asa1997** [comment]
@pramit-d Please resolve the conflicts.

**Comment by pramit-d** [comment]
> @pramit-d Please resolve the conflicts.

@asa1997 resolved. Please check.

File: zaproxy/0.0.1/besman-zaproxy-BT-env.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+{
+    local playbook repo namespace clone_path
+    __besman_check_for_ansible || return 1
+    __besman_check_for_gh || return 1
+    namespace=Be-Secure
+    __besman_check_github_id || return 1
+    __besman_gh_auth "$BESMAN_USER_NAMESPACE"
+    playbook=$HOME/besman-trigger-zaproxy-BT-roles.yml
+    export BESMAN_ANSIBLE_ROLE_PATH=$HOME/tmp
+    export BESMAN_ANSIBLE_GALAXY_ROLES=pramit-d/ansible-role-oah-java:Be-Secure/ansible-role-oah-eclipse
```
Comment (asa1997) [inline zaproxy/0.0.1/besman-zaproxy-BT-env.sh:13]: Please add the package manager as well

Comment (pramit-d) [reply zaproxy/0.0.1/besman-zaproxy-BT-env.sh]: @asa1997 gradle ansible added.

**Comment by pramit-d** [comment]
> @pramit-d Please address the comment

@asa1997 RT BT env updated

**Pull request closed without merging**